lib/output/utils.js

```diff
@@ -36,15 +47,22 @@ function initCtorRegistry(globalObject) {
     ctorRegistry[`%${intrinsic}%`] = globalObject[intrinsic];
   }
 
-  // TODO: Also capture `%AsyncIteratorPrototype%`
   ctorRegistry["%IteratorPrototype%"] = Object.getPrototypeOf(
     Object.getPrototypeOf(new ctorRegistry["%Array%"]()[Symbol.iterator]())
   );
+  ctorRegistry["%AsyncIteratorPrototype%"] = AsyncIteratorPrototype;
 
   globalObject[ctorRegistrySymbol] = ctorRegistry;
   return ctorRegistry;
 }
 
+// Associates a constructor with the given global object. Intended as a public
+// utility for the purpose of registering "%AsyncIteratorPrototype%".
+function registerConstructor(globalObject, key, constructor) {
```

Comment:
The name is a bit imprecise, since we allow registering any intrinsic, not just constructors.